app/scripts/modules/core/src/managed/ManagedResourceStatusIndicator.tsx

```diff
@@ -180,45 +179,61 @@ const LearnMoreLink = ({ resourceSummary }: { resourceSummary: IManagedResourceS
   <a
     target="_blank"
     onClick={() => logClick('Status docs link', resourceSummary.id, resourceSummary.status)}
-    href={`https://www.spinnaker.io/reference/managed-delivery/resource-status/#${resourceSummary.status.toLowerCase()}`}
+    href={`https://www.spinnaker.io/guides/user/managed-delivery/resource-status/#${resourceSummary.status.toLowerCase()}`}
   >
     Learn more
   </a>
 );
 
-const PopoverLinks = ({
+const PopoverActions = ({
   resourceSummary,
   application,
   hidePopover,
 }: {
   resourceSummary: IManagedResourceSummary;
   application: Application;
   hidePopover: () => void;
-}) => (
-  <div className="horizontal right">
-    {!resourceSummary.isPaused && (
-      <p className="sp-margin-m-top sp-margin-xs-bottom">
-        <button className="passive" onClick={() => toggleResourcePause(resourceSummary, application, hidePopover)}>
-          <i className="fa fa-pause" /> Pause management of this resource
-        </button>
-      </p>
-    )}
-    {resourceSummary.isPaused && !application.isManagementPaused && (
-      <p className="sp-margin-m-top sp-margin-xs-bottom">
-        <button className="passive" onClick={() => toggleResourcePause(resourceSummary, application, hidePopover)}>
-          <i className="fa fa-play" /> Resume management of this resource
-        </button>
-      </p>
-    )}
-    {application.isManagementPaused && (
-      <p>
-        <UISref to="home.applications.application.config" params={{ section: 'managed-resources' }}>
-          <a>Resume application management</a>
-        </UISref>
+}) => {
+  const historyButton = (
+    <button
+      className="passive flex-none"
+      onClick={() => {
+        hidePopover();
+        showModal(ManagedResourceHistoryModal, { resourceSummary });
+      }}
+    >
+      <i className="fa fa-history" /> History
+    </button>
+  );
+  return (
+    <div className="horizontal right">
+      <p className="flex-container-h middle sp-margin-m-top sp-margin-xs-bottom sp-group-margin-s-xaxis">
+        {historyButton}
+        {!resourceSummary.isPaused && (
+          <button
+            className="passive flex-none"
+            onClick={() => toggleResourcePause(resourceSummary, application, hidePopover)}
+          >
+            <i className="fa fa-pause" /> Pause management of this resource
+          </button>
+        )}
+        {resourceSummary.isPaused && !application.isManagementPaused && (
+          <button
+            className="passive flex-none"
+            onClick={() => toggleResourcePause(resourceSummary, application, hidePopover)}
+          >
+            <i className="fa fa-play" /> Resume management of this resource
+          </button>
+        )}
+        {application.isManagementPaused && (
+          <UISref to="home.applications.application.config" params={{ section: 'managed-resources' }}>
+            <a>Resume application management</a>
```

Comment:
Since this is displayed in the context of a specific resource, shouldn't there be a warning here that this will affect all resources under the application?

Reply:
this is actually a link to the app config page, where you get deep-linked into the "Managed Resources" section with set of copy/explanation about resuming the application — making that relationship/impact super clear is actually why Chris Berry originally added this as a link, instead of having it do the pause in a modal like with resource pausing.